carts/views.py

```diff
@@ -1,3 +1,40 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http     import JsonResponse
+from django.views    import View
+
+from products.models import Product
+from carts.models    import Cart
+from core.utils      import LoginConfirm
+
+class CartView(View):
+
+    @LoginConfirm
+    def post(self, request):
+        try:
+            data      = json.loads(request.body)
+            product   = Product.objects.get(id = data['product_id'])
+            user      = request.user
+            quantity = data['quantity']
+
+            if not Cart.objects.filter(product=product,user=user).exists():
+                Cart.objects.create(
+                    user     = user,
+                    product  = product,
+                    quantity = quantity
+                )
+            else:
+                temp = Cart.objects.filter(product=product, user=user)[0].quantity
+                print(Cart.objects.filter(product=product, user=user)[0].quantity)
+                temp += quantity
+                Cart.objects.filter(product=product,user=user).update(quantity=temp)
```

Comment:
그러면 이 부분은 이미 장바구니가 존재할 때의 경우가 되는데,
1. temp 처럼 불명확한 변수명은 사용하는 것을 지양합니다.
2. print는 제거해 주세요.
3. 이미 존재 했다고 판단 했으니, get을 통해 해당 장바구니 정보를 가져와서 로직을 구성해 보면 어떨까요?
```suggestion
            else:
                cart = Cart.objects.get(product=product,user=user)
                cart.quantiry += quantity
```